src/localStorage.js

```diff
@@ -44,9 +30,50 @@ export class TaskStorage {
   changeStatus = (e) => {
     const allTasks = this.getTask();
     const form = e.target.checked;
-    const value = e.target.value - 1;
+    const value = e.target.value;
 
     form ? allTasks[value].completed = true : allTasks[value].completed = false;
     localStorage.setItem('alltasks', JSON.stringify(allTasks));
   }
+
+  removeTasks = () => {
+    const allTasks = this.getTask();
+    const tasks = allTasks.filter((task) => {
+      return task.completed != true;
+    });
+
+    console.log(tasksi);
+    localStorage.setItem('alltasks',JSON.stringify(tasks));
+  }
```

Comment:
- [ ] When a todo item is deleted the indexes of the remaining todos should be re-adjusted to be consecutive integers. For example if the initial indexes were 1, 2, 3, 4 and If todo item 3 is deleted the new todos should have indexes of 1, 2, 3 not 1, 2, 4. Kindly fix this. :pray: 

![Screenshot from 2021-12-03 13-06-17](https://user-images.githubusercontent.com/64894927/144584878-817d7732-668f-4529-b84b-b1b401cd1c96.png)